sdk/src/azure/core/az_http_policy_logging.c

```diff
@@ -254,10 +253,13 @@ AZ_NODISCARD az_result az_http_pipeline_policy_logging(
     return _az_http_pipeline_nextpolicy(ref_policies, ref_request, ref_response);
   }
 
-  int64_t const start = az_platform_clock_msec();
+  int64_t start;
```

Comment:
```suggestion
  int64_t start = 0;
```

Reply:
Thank you Rick for finding this! Wow, this is so much not like the regular me :) I usually initialize everything that is initializable, for the consistent behavior.